bigtable/google/cloud/bigtable/table.py

```diff
@@ -386,6 +386,47 @@ def sample_row_keys(self):
             self.name)
         return response_iterator
 
+    def truncate(self, timeout=60):
+        """Truncate the table
+
+        :type timeout: float
+        :param timeout: (Optional) The amount of time, in seconds, to wait
+                        for the request to complete.
+
+        :raise: google.api_core.exceptions.GoogleAPICallError: If the
+                request failed for any reason.
+                google.api_core.exceptions.RetryError: If the request failed
+                due to a retryable error and retry attempts failed.
+                ValueError: If the parameters are invalid.
+        """
+        client = self._instance._client
+        table_admin_client = client._table_admin_client
+        table_admin_client.drop_row_range(self.name,
+                                          delete_all_data_from_table=True,
+                                          timeout=timeout)
+
+    def drop_by_prefix(self, row_key_prefix, timeout=60):
```

Comment:
The timeout here is too low.  Please get the default GAPIC value, test this method and add a system test.